### ISSUE TITLE
Use real MTU size for MTU packets

### ISSUE THREAD

**Issue by xmedeko** [RevenantX/LiteNetLib#243]
Currently, MTU packets `MtuCheck`, `MtuOk` use index to `PossibleMtu` array. It will break if the `PossibleMtu` changes.
Use real size and pls add sizes: (maybe, not so often used)1232, (e.g. Google Cloud) 1460,  and (many VPNs and ISPs) 1472.

**Library version**: 0.8indev

**Comment by SmiggsGDog** [comment]
I do a bit of this in my day job.

Ideally you want to discover the MTU for each connection. Or a quick fix,
let the server set a number and kick anyone that cant comply.

From a consumer perspective I've seen  MTUs as low as 1400 on cellular and
wimax services (very common on these services). 1500 is common for LANs,
fibre and most ADSL services.

Enterprise networks can have larger overhead caused by things like WAN
optimisation devices and/or VPNs or other tunnels. In this case you can see
lower MTUs.

A lower MTU is probably not a big deal for this lib as most people trying
to use it will not be trying to transfer a large amount of data across it.

**Comment by xmedeko** [comment]
Yes, some dynamic MTU discovery may be implemented in the next step.
